tests/system/test_fim/test_synchronization/test_synchronization.py

```diff
@@ -0,0 +1,154 @@
+
+
+# Hosts
+testinfra_hosts = ["wazuh-manager", "wazuh-agent1"]
+
+inventory_path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))),
+                              'provisioning', 'one_manager_agent', 'inventory.yml')
+host_manager = HostManager(inventory_path)
+local_path = os.path.dirname(os.path.abspath(__file__))
+messages_path = [os.path.join(local_path, 'data/messages.yml'),
+                 os.path.join(local_path, 'data/delete_message.yml'),
+                 os.path.join(local_path, 'data/wait_fim_scan.yml'),
+                 os.path.join(local_path, 'data/agent_initializing_synchronization.yml'),
+                 os.path.join(local_path, 'data/manager_initializing_synchronization.yml')
+                 ]
+tmp_path = os.path.join(local_path, 'tmp')
+scheduled_mode = 'testdir1'
+db_path = '/var/ossec/queue/db/001.db'
+db_script = '/var/system_query_db.py'
+
+
+@pytest.mark.parametrize('host', ['wazuh-agent1', 'wazuh-manager'])
+@pytest.mark.parametrize('case', ['add', 'modify', 'delete'])
+@pytest.mark.parametrize('folder_path', ['testdir1'])
+def test_Synchronization_create_file_agent_stopped(folder_path, case, host):
+    '''
+    Description: The test will monitor a directory and apply changes when agent is stopped.
+    Finally, it will verify that the FIM 'Synchronization' event is generated
+    in agent and manager side.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - folder_path:
+            type: str
+            brief: Name of the folder that will be created in the test.
+        - case:
+            type: str
+            brief: Name of the test case that will be created in the test.
+        - host:
+            type: str
+            brief: Name of the endpoint  that will be use in the test.
+
+    assertions:
+        - Verify that FIM sync events are generated correctly on the manager and agent sides.
+    
+    input_description: Different test cases are included with Pytest parametrize.
+                       The test cases are: add, modify and delete files.
+   
+    expected_output:
+        - Different test cases are contained in external YAML file 
+          (agent_initializing_synchronization.yml and manager_initializing_synchronization.yml)
+    tags:
+        - fim_basic_usage
+        - scheduled
+    '''
+    message_path = messages_path[3]
+    # Clear logs, create folder to monitored and restart the service
+    create_folder_file(host_manager, folder_path)
+    host_manager.control_service(host='wazuh-agent1', service='wazuh', state="restarted")
+
+    # Check that the manager contains data to monitor
+    try:
+        HostMonitor(inventory_path=inventory_path,
+                    messages_path=messages_path[0],
+                    tmp_path=tmp_path).run()
+    except:
+        host_manager.run_command('wazuh-agent1', f'rm -rf {folder_path}')
+
+    clean_logs(host_manager)
+
+    # Stop agent
```

Comment:
There are 2 cases: the agent stops and the manager stops.

Reply:
Done in commit [1321a](https://github.com/wazuh/wazuh-qa/pull/2536/commits/1321a0318f984b93edcdab0ec1d3ec9854c420f4)